pulsar-client/src/main/java/org/apache/pulsar/client/impl/MessageCrypto.java

```diff
@@ -511,7 +529,8 @@ private ByteBuf decryptDataKeyAndData(MessageMetadata msgMetadata, ByteBuf paylo
                     break;
                 }
             } catch (Exception e) {
-                // First time, entry won't be present in cache 
+                // First time, entry won't be present in cache
```

Comment:
Why should we get an exception if it's not in cache? Exceptions are expensive, should we just check for null before?